### PR TITLE
Include keyboard

### PR DESCRIPTION
**Pull request by lavolp3**
This is a PR to include a keyboard to add items to the list. Instead of coding a keyboard into this module I added an "Add" button and a communication channel to the module [MMM-Keyboard](https://github.com/lavolp3/MMM-Keyboard) which would have to be installed separately.
Please have a look at it. If you like it I can complete the documentation in the README. 

image

### CODE REVIEW

**Comment by lavolp3** [comment]
would close issue #8

**Comment by werthdavid** [comment]
Awesome thank you I’ll take a look after the holidays! Much appreciated!

File: MMM-Bring.js [hunk 1]
```diff
@@ -127,6 +140,11 @@ Module.register("MMM-Bring", {
         return container;
     },
 
+    openKeyboard: function() {
+        console.log("MMM-Bring opening keyboard");
```
Comment (werthdavid) [inline MMM-Bring.js:144]: Should we use logger instead?

File: MMM-Bring.js [hunk 2]
```diff
@@ -135,6 +153,18 @@ Module.register("MMM-Bring", {
             this.sendSocketNotification("GET_LIST", this.config);
         }
     },
+    
+    notificationReceived: function(notification, payload) {
+        if (notification === "KEYBOARD_INPUT" && payload.key === "bring" && payload.message != '') {
```
Comment (werthdavid) [inline MMM-Bring.js:158]: payload.key could be "more unique" with mmm-bring but this one is fine as well

File: MMM-Bring.js (hunk 1 above)
Comment (lavolp3) [reply MMM-Bring.js:144]: Sorry, What do you mean with logger?

I usually use a debug logger function in my modules:
```
  log: function (msg) {
    if (this.config && this.config.debug) {
      console.log(this.name + ": ", (msg));
    }
  }
```
Can implement this here if you like.

File: MMM-Bring.js (hunk 2 above)
Comment (lavolp3) [reply MMM-Bring.js:158]: Changed to "mmm-bring"

File: MMM-Bring.js (hunk 1 above)
Comment (lavolp3) [reply MMM-Bring.js:144]: Or do you mean `log.info(msg)`?

Comment (werthdavid) [reply MMM-Bring.js:144]: There is a Log-Object available for logging.
e.g. Log.info("test") so the log-output appears in the MM-log and not the browser window. What do you say? console.log is also fine for me, it's just that I've seen other projects using this instead

Comment (lavolp3) [reply MMM-Bring.js:144]: I thought `Log.info` goes into stderr and hence into the devtools as well as `console.log`?
https://github.com/MichMich/MagicMirror/wiki/Module-Debugging#logging-from-mmm-your-modulejs.

But I can change it of course. It's your repository :-)

Comment (lavolp3) [reply MMM-Bring.js:144]:
```
var Log = (function() {
	return {
		info: Function.prototype.bind.call(console.info, console),
		log:  Function.prototype.bind.call(console.log, console),
```

Comment (werthdavid) [reply MMM-Bring.js:144]: Ok good to know thanks...

**Comment by werthdavid** [comment]
Thanks for the work!

**Pull request merged**